src/components/points-of-interest/PointOfInterest.jsx

```diff
@@ -51,7 +52,9 @@ function PointOfInterest({
       );
       setProximity(distance);
       if (!unlocked && id === nextUnlockableId) {
-        setUnlocked(distance < accuracy);
+        setUnlocked(
+          distance < accuracy || distance < Number(proximityToUnlock)
```

Comment:
`proximityToUnlock` should already be a number, cf. https://github.com/itk-dev/aapodwalk_api/pull/10#discussion_r1295596262.